### PR TITLE
Add more styling to card and conditional rendering to card

### PR DESCRIPTION
**Pull request by ashleylevi**
Added styling to our card component.
Added conditional rendering to our card component to display name of show based on what show the episode is from.

@brittanystoroz
@ThatPamIAm

### CODE REVIEW

File: src/Card.js [hunk 1]
```diff
@@ -19,19 +19,40 @@ export default class Card extends Component {
 
 
   render() {
+    console.log('starring', this.props.episode.starring)
```
Comment (KylieStanley) [inline src/Card.js:22]: I like that you are checking to see what information you are getting, but we may want to remove this console.log before merging it all in so we don't miss it later.

File: src/Card.js [hunk 2]
```diff
@@ -19,19 +19,40 @@ export default class Card extends Component {
...
+    if (this.props.episode.starring.includes('Willow')) {
```
Comment (KylieStanley) [inline src/Card.js:23]: Nice use of conditional logic for rendering the show titles!

File: src/Card.js [hunk 3]
```diff
@@ -19,19 +19,40 @@ export default class Card extends Component {
...
+          <i className="fas fa-plus" onClick={this.handleState}></i>
+        </div>
+      </li>
+    )
+
   }
-}
\ No newline at end of file
+}
+}
```
Comment (KylieStanley) [inline src/Card.js:58]: I like the if/else, but there is also quite a bit being repeated. Maybe we can try and dry this up a bit later.

File: public/index.html [hunk 4]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
```
Comment (K-Rye) [inline public/index.html:7]: This looks clean, fits the design layout. Might need to play with size and or color as we move forward.

File: src/Card.js [hunk 5]
```diff
@@ -19,19 +19,40 @@ export default class Card extends Component {
...
           <p>{this.props.episode.synopsis}</p>
           <i className="fas fa-plus" onClick={this.handleState}></i>
         </div>
       </li>
-    );
+    )
+  } else {
+    return (
+      <li className="card">
+      <div className="background" style={{ backgroundImage: `url(${this.props.episode.image})` }}></div><h3 className="episode-name">
```
Comment (brittanydionigi) [inline src/Card.js:42]: The line spacing here is a little weird, I'd put that h3 on it's own line along with the text inside of it

`<h3>{this.props.episode.name}</h3>`

**Pull request merged**